src/api/util/helpers.ts

```diff
@@ -0,0 +1,33 @@
+import { DbMetadataLocaleBundle } from '../../pg/types';
+import { MetadataType } from '../types';
+
+export function parseMetadataLocaleBundle(
+  locale?: DbMetadataLocaleBundle
+): MetadataType | undefined {
+  let response: MetadataType | undefined;
+  if (locale && locale.metadata) {
+    response = {
+      sip: locale.metadata.sip,
+      name: locale.metadata.name ?? undefined,
+      description: locale.metadata.description ?? undefined,
+      image: locale.metadata.image ?? undefined,
+    };
+    if (locale.attributes.length > 0) {
+      response.attributes = locale.attributes.map(item => ({
+        trait_type: item.trait_type,
+        value: JSON.parse(item.value),
```

Comment:
Any reason for storing the JSON in pg as `text` instead of one of the json types? Could imagine a future query that leverages pg json query capability to search this object